portal/src/graphql/adminapi/UsersList.tsx

```diff
@@ -289,7 +289,10 @@ const UsersList: React.VFC<UsersListProps> = function UsersList(props) {
         default:
           return (
             <div className={styles.cell}>
-              {item[column?.key as keyof UserListItem] ?? USER_LIST_PLACEHOLDER}
+              <div className={styles.cellText}>
+                {item[column?.key as keyof UserListItem] ??
+                  USER_LIST_PLACEHOLDER}
+              </div>
```

Comment:
Can we remove this extra div and write `cn(styles.cell, styles.cellText)` instead?

Reply:
`text-overflow: ellipsis` doesn't work with `display: flex`, so the extra div is needed.